utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -369,6 +369,10 @@ int main(int argc, char **argv)
     if(testCase == 46)
         CHECK_RETURN_STATUS(hipHostMalloc(&intensity, batchSize * sizeof(Rpp32f)));
 
+    RpptChannelOffsets *rgbOffsets;
+    if(testCase == 35)
+        CHECK_RETURN_STATUS (hipHostMalloc (&rgbOffsets, batchSize * sizeof(RpptChannelOffsets)));
```

Comment:
Please remove the additional space after CHECK_RETURN_STATUS, hipHostMalloc

Reply:
Done